core/transformer/src/test/standalone/IModelTransformer.test.ts

```diff
@@ -2126,7 +2126,7 @@ describe("IModelTransformer", () => {
     newDb.close();
   });
 
-  it("transforms code values with non standard space characters", async () => {
+  it.skip("transforms code values with non standard space characters", async () => {
```

Comment:
want to confirm this is meant to be skipped

Reply:
@MichaelBelousov, should I unskip this?

Reply:
He's on vacation til tomorrow. I will resolve this issue to prevent blocking this pr; will open an issue to make sure we track this.